### DISCUSSION TITLE
Why I got regulons with fewer than 20 genes?

### DISCUSSION THREAD

**Discussion by Sophia409** [aertslab/SCENIC#269] [Python] [answered]
Hello, aertslab

Since all regulons targeting fewer than 20 genes are dropped (in the modules_from_adjacencies function, why I got regulons with fewer than 20 genes?
![image](https://user-images.githubusercontent.com/30619010/71080565-5e731e80-21c8-11ea-8485-6868c80b4861.png)

Hope for your reply

**Comment by liuyifang** [comment]
Hi, I am also curious about this question.

**Comment by Sophia409** [comment]
@s-aibar

**Comment by cflerin** [comment, marked as answer]
`modules_from_adjacencies` is a pySCENIC function, but in there, the 20 gene threshold is for the creation of the coexpression modules. Once these modules are pruned into the final regulons, it's possible for the regulon to have fewer than 20 genes.
